content/en/blog/_posts/2023-01-17-confidential-kubernetes.md

```diff
@@ -0,0 +1,312 @@
+and can do so with relatively low overhead. These technologies each have different design goals,
+advantages, and limitations, and they are available in different environments including consumer
+laptops, servers, and mobile devices.
+
+Additionally, we should mention
+[ARM TrustZone](https://www.arm.com/technologies/trustzone-for-cortex-a) which is optimized
+for embedded devices such as smartphones, tablets, and smart TVs, as well as
+[AWS Nitro Enclaves](https://aws.amazon.com/ec2/nitro/nitro-enclaves/) which are only available
+on [Amazon Web Services](https://aws.amazon.com/) and have a different attacker model compared
+to the CPU-based solutions by Intel and AMD.
+
+### Security properties and feature set
+
+In the following, we will review the security properties and additional features these
+new technologies bring to the table. Not all solutions will provide all properties, and
+we will discuss each technology in further detail in their respective chapter.
+
+The **Confidentiality** property ensures that information cannot be viewed while it is
+in use in the TEE. This provides us with the highly desired feature to secure
+**data in use**. Depending on the specific TEE used, both code and data may be protected
+from outside viewers.
+
+Confidentiality is a great feature, but an attacker would still be able to manipulate
+or inject arbitrary code and data for the TEE to execute and therefore easily leak critical
+information. **Integrity** guarantees a TEE owner that neither code nor data can be
+tampered with while running critical computations.
+
+*Availability* is the third basic property often discussed in the context of information
+security. This property is outside the scope of most TEEs. Usually, they can be controlled
+(shut down, restarted, …) by some higher-level concept. This could be the CPU itself, a
+hypervisor, or a kernel. This is to preserve the availability of the overall system, and
+not the TEE itself. When running in the cloud the availability is usually guaranteed by
+the cloud provider in terms of Service Level Agreements (SLAs) and is not cryptographically enforceable.
+
+Confidentiality and Integrity by themselves would only be helpful in some cases. For example,
+consider a TEE running in a remote cloud. How would you know that the TEE is genuine and running
+your intended software? It could be an imposter stealing your data as soon as you send it over.
+This fundamental problem is addressed by **Attestability**. Attestation allows us to verify
+the identity, confidentiality, and integrity of TEEs based on cryptographic certificates issued
+from the hardware itself. This feature can also be made available to clients outside of the
+confidential computing hardware in the form of remote attestation.
+
+TEEs can hold and process information that needs to persist even beyond their lifespan. That
+could mean across restarts, different versions, or platform migrations. Therefore **Recoverability**
```

Comment:
I would like to keep as is, since this list is not meant to be exhaustive, but only hint at different concepts. Pause and restart are similar in that regard.